### PR TITLE
TRUNK-5151: Remove body of voidRelationship

### PR DESCRIPTION
**Pull request by hillarykimo**
**Description of what I changed**
I removed

if (relationship.getVoided()) {
			
return relationship;
		}
		
		relationship.setVoided(true);
		if (relationship.getVoidedBy() == null) {
			relationship.setVoidedBy(Context.getAuthenticatedUser());
		}
		if (voidReason != null) {
			relationship.setVoidReason(voidReason);
		}
		relationship.setDateVoided(new Date());
		
		return Context.getPersonService().saveRelationship(relationship);
which is already being called via AOP the Relationship and fixed the methods to effectively return null.

**Issue I worked on**
https://issues.openmrs.org/browse/TRUNK-5151

### CODE REVIEW

**Comment by sherrif10** [comment]
@hillarykimo  Could you respond to the above reviews.

**Comment by hillarykimo** [comment]
@sherrif10  I believe your saw my post on Talk its for that reason cause every time i push it show that error but am still trying to find out why

**Comment by sherrif10** [comment]
@kdaud  i seem to have responded to the talk thread. Did  you respond on talk thread.

**Comment by hillarykimo** [comment]
@sherrif10  I tried the commands but it still failing thou am finding other ways now and trying to see why they are failing

**Comment by hillarykimo** [comment]
@sherrif10 and @ibacher  https://github.com/openmrs/openmrs-core/commit/919d076615a4dd1c81da2862252dfdaa5d95ccd6 I finally managed to push please check it out .

File: api/src/main/java/org/openmrs/api/impl/PersonServiceImpl.java [hunk 1]
```diff
@@ -488,18 +488,11 @@ public Relationship saveRelationship(Relationship relationship) throws APIExcept
...
-		}
-		
-		relationship.setVoided(true);
-		if (relationship.getVoidedBy() == null) {
-			relationship.setVoidedBy(Context.getAuthenticatedUser());
-		}
-		if (voidReason != null) {
-			relationship.setVoidReason(voidReason);
+		if(relationship == null) {
+
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/api/impl/PersonServiceImpl.java:492]: Do you need this extra space ? at lines 492 and 494

Comment (hillarykimo) [reply api/src/main/java/org/openmrs/api/impl/PersonServiceImpl.java]: @tendomart  https://github.com/openmrs/openmrs-core/pull/3890/commits/92a512760f8459668a5af3145777701bb776bd86 hope this is right

**Comment by tendomart** [comment]
Let's wait for the build .Hoping you did run locally as well

**Pull request merged**